### PR TITLE
Support comma separated string include in BindAttribute

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Issue - #2564 
@rynowak @Eilon

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs [hunk 1]
```diff
@@ -136,5 +140,17 @@ private ObjectFactory GetFactory()
...
         }
+
+        private static string[] SplitString(string original)
+        {
+            if (string.IsNullOrEmpty(original))
+            {
+                return new string[0];
+            }
+
+            var split = original.Split(',').Select(piece => piece.Trim()).Where(piece => !string.IsNullOrEmpty(piece));
```
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs]: Look at the overload of `Split` with `StringSplitOptions` - then you don't need any of this ceremony

File: src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs [hunk 2]
```diff
@@ -136,5 +140,17 @@ private ObjectFactory GetFactory()
...
+
+            return split.ToArray();
```
Comment (rynowak) [inline src/Microsoft.AspNet.Mvc.Core/BindAttribute.cs]: A bit of a nitpick, but think about removing some of the ToArrays here by just adding to a list or using `SelectMany` in your top level loop.

**Comment by Eilon** [comment]
:shipit:

**Comment by rynowak** [comment]
:shipit:

**Pull request merged**